src/rtext.c

```diff
@@ -1415,7 +1415,7 @@ int TextCopy(char *dst, const char *src)
         return 0;
 
     // use strcpy since it uses vector operations
-    unsigned int length = strcpy(dst, src);
+    unsigned int length = *strcpy(dst, src);
```

Comment:
@DissolveDZ I'm sorry, my suggestion was wrong. Tested it again now and it doesn't return the the correct string length. I believe this would be the correct way:
```
unsigned int length = strlen(strcpy(dst, src));
```